pub fn handle_error() {
    panic!("Errrroorrr....");
}

//pub fn handle_error() -> Result<T, E> {
//    println!("handle_error");
//
//    Ok(())
//}
